pkg/server/server.go

```diff
@@ -293,3 +298,15 @@ func writeToken(token, file, certs string) error {
 	token = FormatToken(token, certs)
 	return ioutil.WriteFile(file, []byte(token+"\n"), 0600)
 }
+
+func setNoProxyEnv(config *config.Control) error {
+	envList := strings.Join([]string{
+		os.Getenv("NO_PROXY"),
+		config.ClusterIPRange.String(),
+		config.ServiceIPRange.String(),
+	}, ",")
+	if err := os.Setenv("NO_PROXY", envList); err != nil {
+		return err
+	}
+	return os.Setenv("no_proxy", envList)
```

Comment:
We should probably either leave the lowercase `no_proxy` unmodified or append to the current value to prevent overwriting user values.